src/types.ts

```diff
@@ -73,7 +73,7 @@ export interface ExportProps {
 
 export interface BlendProps {
     opacity?: number;
-    blendMode?: BlendMode;
+    blendMode?: 'PASS_THROUGH' | BlendMode;
```

Comment:
This was also required to make the `blendMixin` happy and copies the definition from:

https://github.com/figma/plugin-typings/blob/master/index.d.ts#L539